index.md

```diff
@@ -87,18 +87,18 @@ document.addEventListener('DOMContentLoaded', function() {
 ### Teaching Assistants
 
 * Name: Chandrahas Aroori
-   * Office Hours: TBD
-   * Office Hours Location: CCDS TBD
+   * Office Hours: Th 11:00am - 12:00pm and by appointment
+   * Office Hours Location: CCDS 1623
```

Comment:
That's my office! :-) suggest holding them on 15th floor at the open desks on the east wall. We should double check but should change Farid's to that too.

Reply:
Actually that's my office hours time too. Chandrahas should pick a different time.